### PR TITLE
ENTESB-15825 Kameletbinding with knative for AWS-SQS, Salesforce, Telegram

### PR DESCRIPTION
**Pull request by lfabriko**

### CODE REVIEW

File: test/aws-sqs/aws-sqs-to-inmem.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    properties:
+      queueNameOrArn: ${camel.kamelet.aws-sqs-source.aws-sqs-credentials.queueNameOrArn}
+      accessKey: ${camel.kamelet.aws-sqs-source.aws-sqs-credentials.accessKey}
+      secretKey: ${camel.kamelet.aws-sqs-source.aws-sqs-credentials.secretKey}
+      region: ${camel.kamelet.aws-sqs-source.aws-sqs-credentials.region}
+  sink:
+    ref:
+      kind: InMemoryChannel
+      apiVersion: messaging.knative.dev/v1
+      name: messages
```
Comment (vkasala) [inline test/aws-sqs/aws-sqs-to-inmem.yaml:37]: Add a new line Lucie.

File: test/salesforce/salesforce-to-inmem.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      clientId: "${camel.kamelet.salesforce-source.salesforce-credentials.clientId}"
+      clientSecret: "${camel.kamelet.salesforce-source.salesforce-credentials.clientSecret}"
+      userName: "${camel.kamelet.salesforce-source.salesforce-credentials.userName}"
+      password: "${camel.kamelet.salesforce-source.salesforce-credentials.password}"
+      loginUrl: "${camel.kamelet.salesforce-source.salesforce-credentials.loginUrl}"
+  sink:
+    ref:
+      kind: InMemoryChannel
+      apiVersion: messaging.knative.dev/v1
+      name: messages
```
Comment (vkasala) [inline test/salesforce/salesforce-to-inmem.yaml:23]: Add a new line at the end of the file.

File: test/telegram/telegram-to-inmem.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      kind: Kamelet
+      apiVersion: camel.apache.org/v1alpha1
+      name: telegram-source
+    properties:
+      authorizationToken: ${camel.kamelet.telegram-source.telegram-credentials.authorizationToken}
+  sink:
+    ref:
+      kind: InMemoryChannel
+      apiVersion: messaging.knative.dev/v1
+      name: messages
```
Comment (vkasala) [inline test/telegram/telegram-to-inmem.yaml:34]: Add a new line at the end of the file.

**Review by christophd — COMMENTED** [review]
LGTM. Please add new line to files as suggested by @vkasala and it should be ok to merge

**Pull request merged**